nifty-core/src/main/java/com/facebook/nifty/ssl/SSLDefaults.java

```diff
@@ -0,0 +1,27 @@
+package com.facebook.nifty.ssl;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class SSLDefaults {
+    public static final List<String> SERVER_DEFAULTS;
+
+    static {
+        SERVER_DEFAULTS = new ArrayList<String>();
```

Comment:
Should we use `ImmutableList` for these?
Also, where does the list come from? It makes total sense, but should we update it when new releases of OpenSSL are made? Or when vulnerabilities are discovered? Does OpenSSL have a "good" list we can reference rather than copy?